api/types/v1alpha1/srl_version_test.go

```diff
@@ -63,6 +63,11 @@ func TestParseVersionString(t *testing.T) {
 			got:  "0.0.0-34652",
 			want: &SrlVersion{"0", "0", "0", "34652", ""},
 		},
+		{
+			desc: "version_0.0.0-34652",
+			got:  "version_0.0.0-34652",
+			want: &SrlVersion{"0", "0", "0", "34652", ""},
+		},
 		{
 			desc: "latest",
 			got:  "latest",
```

Comment:
I would like to see the following usecases be met:

1) If a user specifies a model that does not require a license then the tag of their image should not matter

2) If a user specifies a model that does require a license and the user has provided a single "catch all" license then the tag of their image should not matter

3) If a user specifies a model that does require a license and the user has not provided a license or has provided a license that matches a specific version other than the one they provided then the failure should clearly output during topology creation instead of the current behavior of hanging (presumably due to some wait or retry from kne <-> srl-controller) or bringing the pods up, marking them running, then realizing there is no license and continue into a crash loop

What work would you need on KNE to help achieve this in tandem with srl-controller changes? Do the above usecases seem like the right behavior to you?

Reply:
Also in the short term can you add the "ga" tag as recognized in the same way "latest" is?